### PR TITLE
Tutorial pop up design

### PR DESCRIPTION
**Pull request by kcirerick**
Creates the Html and CSS for the popup to be displayed on the landing page.

### CODE REVIEW

File: src/main/webapp/tutPopUp.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tutorial PopUp</title>
+    <link rel="stylesheet" href="style.css"></link>
```
Comment (ccjeane) [inline src/main/webapp/tutPopUp.html:6]: tutPopUp.css?

File: src/main/webapp/tutPopUp.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
```
Comment (ccjeane) [inline src/main/webapp/tutPopUp.html:8]: We don't have a script.js file - all have more specific names.

Comment (dterr) [reply src/main/webapp/tutPopUp.html:8]: [We should use a lowercase name.](https://google.github.io/styleguide/jsguide.html#file-name)
~~~tutorial-populator.js

File: src/main/webapp/tutPopUp.html (hunk 1 above)
Comment (kcirerick) [reply src/main/webapp/tutPopUp.html:6]: good catch!

File: src/main/webapp/tutPopUp.html (hunk 2 above)
Comment (kcirerick) [reply src/main/webapp/tutPopUp.html:8]: i'm on that wave, we don't need any js for this file actually, I had just copied it from index.html, resolved now.

File: target/portfolio-1/tutPopUp.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tutorial PopUp</title>
+    <link rel="stylesheet" href="tutPopUp.css"></link>
+    <script src="script.js"></script>
```
Comment (ccjeane) [inline target/portfolio-1/tutPopUp.html:7]: script.js doesn't exist

File: src/main/webapp/tutorial-pop-up.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tutorial PopUp</title>
+    <link rel="stylesheet" href="tutPopUp.css"></link>
+    <script src="script.js"></script>
```
Comment (ccjeane) [inline src/main/webapp/tutorial-pop-up.html:7]: script.js doesn't exist

File: src/main/webapp/tutorial-pop-up.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Tutorial PopUp</title>
+    <link rel="stylesheet" href="tutorial-pop-up.css"></link>
+    <script src="indexScript.js"></script>
+  </head>
+  <body>
+      <div id="popUp">
+          <div id="header">
+          	  <div id="exit">x</div>
```
Comment (ccjeane) [inline src/main/webapp/tutorial-pop-up.html:12]: Indented too far in

File: target/portfolio-1/index-script.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+
+// Creates a map centered at the Googleplex!
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422, lng: 122.0841}, zoom: 13, mapTypeId: 'satellite'});
```
Comment (ccjeane) [inline target/portfolio-1/index-script.js:6]: Pull from dev so this negates to get rid of merge issues

File: target/portfolio-1/index-style.css [hunk 7]
```diff
@@ -0,0 +1,5 @@
+#map {
+  border: thin solid black;
```
Comment (ccjeane) [inline target/portfolio-1/index-style.css:2]: pull from dev so this file isn't here

Comment (kcirerick) [reply target/portfolio-1/index-style.css:2]: not entirely sure what you mean here, but pulled from dev on most recent push, hope that resolves this.

File: src/main/webapp/tutorial-pop-up.html (hunk 5 above)
Comment (kcirerick) [reply src/main/webapp/tutorial-pop-up.html]: i'll be more careful

**Pull request merged**